src/pages/Allergens/Allergens.tsx

```diff
@@ -11,23 +11,27 @@ export const AllergensPage = (): ReactElement => {
   const { products } = useProducts();
   const navigate = useNavigate();
 
+  const filteredProducts = useMemo(() => {
+    return products.data?.filter((product) => !product.retired) ?? [];
+  }, [products.data?.length]);
+
   return (
     <PageLayout title={'Allergens - Answer King'}>
-      <div className="container mx-auto mt-10 mb-10 w-[50%] min-w-[500px]">
-        <p className="mb-5 border-b-2 border-[#5A6675] p-5 text-center text-[36px] font-light">Allergen Info</p>
+      <div className="container mx-auto mt-10 mb-10 md:w-[90vw] lg:w-[75vw] xl:w-[50vw]">
+        <p className="mb-5 p-5 text-center text-[36px] font-light">Allergen Info</p>
 
-        {products.data?.length ? (
-          <AllergenTable products={products.data} allergens={ALLERGENS} />
-        ) : (
-          <p className="mb-5 p-5 text-center font-light">No items found</p>
-        )}
+        <div className="rounded-xl bg-white p-[24px] text-[#333F4C]">
+          {filteredProducts.length ? (
```

Comment:
Might want to write a component test for this scenario - see: OrderPanel.cy.tsx for this.

I think you will need to add a retired product to component-test-data and assert that it is not visible.

Happy to write this if you want.